probe-rs/src/debug/snapshots/probe_rs__debug__debug_info__test__RP2040__full_unwind.snap

```diff
@@ -365,8 +365,8 @@ expression: stack_frames
                     - name:
                         Named: data_ptr
                       type_name:
-                        Pointer: ~
-                      value: "<referenced type> @ 0x20000068"
+                        Pointer: u8
+                      value: "*raw u8 @ 0x20000068"
```

Comment:
I just picked it as a generic "raw pointer" because we don't really know the mutability. Easy enough to change to anything else, hard to know what to change to IMO.

Reply:
Looking at it, if we get too specific, we are going have a term that doesn't apply to one or other of Rust vs C.

I like the fact that you (@bugadani ) brought the destination data type up to this level. How does everyone feel about a combo of old and new, with something like "pointer to u8 @ ...", or "reference to u8 @ ...", or ????

Reply:
This is very much Rust-specific and also slice-specific as far as I've seen it. The constness is dependent on whether the slice type is mutable or not, but rustc/llvm does not emit the usual pointer information. Complicating the output further seems counterproductive to me (you'd still have 3 pointer variants, one of them just wouldn't look like the others), but if we can guess based on the parent variable, that may be good enough.

Reply:
Yeah, I don't have a better suggestion, and am OK to leave it as is until someone offers one. It is one of the things I appreciate about open source communities ... if there is something I don't know, then there is usually someone who does ;-)